frontend/src/metabase/dashboard/containers/AutomaticDashboardApp.jsx

```diff
@@ -146,17 +146,15 @@ class AutomaticDashboardApp extends React.Component {
           </div>
           {more && (
             <div className="flex justify-end px4 pb4">
-              {more.map(item => (
-                <Link
```

Comment:
why are we dropping the more.map line here?

Reply:
I've slightly changed the response: :more is no longer part of :related and is just an url string instead of a map (see https://github.com/metabase/metabase/pull/7707/files#diff-21ec7c06c7c922599febe23e7d679142L635). I was needlessly complicating things before and semantics of more changed when we moved it out of the sidebar.